webapp/app/static/js/field_utils.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+function assignAssociatedContent(selector='label, .example-input, .additional-input-content, .invalid-feedback') {
+  let associated_content = document.querySelectorAll(selector);
+    for (let i = 0; i < associated_content.length; i++) {
+      if (associated_content[i].getAttribute('for') != '') {
+         let assignedElement = document.getElementById(associated_content[i].getAttribute('for'));
+         if (assignedElement){
+          if (assignedElement.associatedContent){
+            assignedElement.associatedContent.push(associated_content[i]);
+          } else {
+            assignedElement.associatedContent = [associated_content[i]];
+          }
+         }
+      }
+    }
+}
+function deleteSelectOptions(selectInput, defaultOptionText) {
+  while (selectInput.options.length) {
+    selectInput.remove(0);
+    }
+  selectInput.options.add(new Option(defaultOptionText, ""));
+}
+
+function deleteInput(input){
+  if (input.type == "checkbox"){
+    input.checked = false;
+  } else {
+    input.value = '';
+  }
+}
+function hideInput(input) {
+  input.style.setProperty("display", "none", "important");
+  if (input.associatedContent) {
+    input.associatedContent.forEach((associatedElem) =>{
+      associatedElem.oldDisplayStyle = "inline-block";
+      associatedElem.style.setProperty("display", "none", "important");
+
```

Comment:
Curious: did you add "important" because you found it not to work without? Inline styles have a very high specificity, so it shouldn't be needed?

Reply:
Yes, I had to add it because it did not work otherwise. The problem is that some of the classes use important and therefore overwrites everything else.